draft-tschofenig-rats-psa-token.md

```diff
@@ -663,8 +660,43 @@ would include a policy with appraisal for the following claims:
   locate security certification information associated with the attesting
   device. An example could be a reference to a {{PSACertified}} certificate.
 
-The protocol used to convey Endorsements and Reference Values to the Verifier
-is not in scope for this document.
+## AR4SI Trustworthiness Claims Mappings
+
+{{RATS-AR4SI}} defines an information model that Verifiers can employ to
+produce Attestation Results.
+AR4SI provides a set of standardized appraisal categories and tiers that
+greatly simplifies the task of writing Relying Party policies in multi-attester
+environments.
+
+The contents of {{tab-ar4si-map}} are intended as guidance for implementing a
+PSA Verifier that computes its results using AR4SI.
+The table describes which PSA Evidence claims (if any) are related to which
+AR4SI trustworthiness claim, and therefore what the Verifier must consider when
+deciding if and how to appraise a certain feature associated with the PSA
+Attester.
+
+Trustworthiness Vector claims | Related PSA claims
+---|---
+`configuration` | Software Components ({{sec-sw-components}})
+`executables` | ditto
+`file-system` | N/A
+`hardware` | Implementation ID ({{sec-implementation-id}})
+`instance-identity` | Instance ID ({{sec-instance-id-claim}}).  The Security Lifecycle ({{sec-security-lifecycle}}) can also impact the derived identity.
+`runtime-opaque` | Indirectly derived from `executables`, `hardware`, and `instance-identity`.  The Security Lifecycle ({{sec-security-lifecycle}}) can also be relevant: for example, any debug state will expose otherwise protected memory.
+`sourced-data` | N/A
+`storage-opaque` | Indirectly derived from `executables`, `hardware`, and `instance-identity`.
+{: #tab-ar4si-map title="AR4SI Claims mappings"}
+
+This document does not prescribe what value must be chosen based on each
+possible situation: when assigning specific Trustworthiness Claim values, an
+implementation is expected to follow the algorithm described in {{Section 2.3.3
+of RATS-AR4SI}}.
+
+## Endorsements, Reference Values and Verification Key Material
+
+{{PSA-Endorsements}} defines a protocol based on the CoRIM data model that can
```

Comment:
Does CoRIM need qualifying (referencing)?

Reply:
good point, will add ref.

Reply:
done in [0e15d69](https://github.com/thomas-fossati/draft-psa-token/pull/76/commits/0e15d69f33e33723faab7e8bcd123a7605af753b)